net-voip/sipvicious/sipvicious-0.3.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="SIPVicious is a SIP security package"
+HOMEPAGE="https://github.com/EnableSecurity/sipvicious/wiki"
+SRC_URI="https://github.com/EnableSecurity/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="$(python_gen_cond_dep '
+		dev-python/dnspython[${PYTHON_USEDEP}]
+		dev-python/reportlab[${PYTHON_USEDEP}]
+	')"
+
+src_prepare() {
+	default
+	sed -re 's:man/man1:share/man/man1:' -i setup.py || die "Error updating man page installation location."
```

Comment:
Optional but personally think an elaborate `die` message is not needed for a self-explanatory sed (the line will be displayed on error), so just `|| die` is fine.

This also doesn't use extended regex, so -r is superfluous.

Reply:
I still prefer keeping the text, what you consider self-explanatory may not be for others.